lighthouse-core/test/computed/metrics/lantern-largest-contentful-paint-test.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const assert = require('assert');
+
+const trace = require('../../fixtures/traces/lcp-m79.json');
+const devtoolsLog = require('../../fixtures/traces/lcp-m79.devtools.log.json');
+const LanternLargestContentfulPaint =
+    require('../../../computed/metrics/lantern-largest-contentful-paint.js');
+
+/* eslint-env jest */
+describe('Metrics: Lantern LCP', () => {
+  it('should compute predicted value', async () => {
+    const settings = {};
+    const computedCache = new Map();
+    const result = await LanternLargestContentfulPaint.request(
+      {trace, devtoolsLog, settings},
+      {computedCache}
+    );
+
+    expect({
+      timing: Math.round(result.timing),
+      optimistic: Math.round(result.optimisticEstimate.timeInMs),
+      pessimistic: Math.round(result.pessimisticEstimate.timeInMs),
+    }).toMatchInlineSnapshot(
+      {},
+      `
+Object {
+  "optimistic": 17637,
+  "pessimistic": 17637,
+  "timing": 17637,
```

Comment:
😮  just noting that that 17s is a pretty huge LCP

Reply:
in the trace this is at 1750ish ms

![image](https://user-images.githubusercontent.com/39191/68536791-73c57500-030d-11ea-8386-904c7c96521d.png)


obviously this lantern lcp isn't perfect but i wouldnt expect that we'd have a 10x diff between observed and simulated.

would you mind looking into why it's so high?